### PR TITLE
Write November 2019 newsletter

### PR DESCRIPTION
**Pull request by malvikasharan**
# Summary

The first draft of Nov's newsletter.  When approved will upload images as well. 

Draft with the images is here: https://docs.google.com/document/d/1cs1y5GsMM_QC9Y6BADT0AHttoVuap-lL2dshxky2OFA/edit

### List of changes proposed in this PR (pull-request)

- [x] first draft of the newsletter
- [x] TW Expansion announcement
- [x] Collab cafe dates
- [x] Highlights from the community

### What should a reviewer concentrate their feedback on?

- [x] Approve Content
- [x] Approve language and typos
- [x] Propose changes or add content
- [ ] Expansion sprint report draft

@KirstieJane can you please review this draft?

### CODE REVIEW

File: communications/newsletters/newsletter_12_Nov2019.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+We are excited to announce that *The Turing Way* is expanding into [a series of books](https://github.com/alan-turing-institute/the-turing-way).
+In addition to its previous focus on reproducibility, this project will aim to capture the vast amounts of knowledge of data science practices in the areas of project design, communication, outreach and ethics.
+Following an open source, community driven approach, we will welcome contributions from anyone from within the Turing community and beyond  who can support researchers and practitioners in learning or improving the "how" of conducting world-leading research in data science.
+The project proposal with further details is available [online](https://github.com/alan-turing-institute/the-turing-way/blob/master/project_management/tps-funding-application-20190429.md).
+
+To plan the next steps, the core team met last month on 11 and 12 October to brainstorm and discuss the next steps in this project.
+The participants of this meeting explored a number of topics in four themed sessions, draft report from which is available on [this HackMD](https://hackmd.io/zVTeKhG2SIiBLam1YtILNg).
+
+![]()
```
Comment (KirstieJane) [inline communications/newsletters/newsletter_12_Nov2019.md]: Which picture are you thinking of here?

File: communications/newsletters/newsletter_12_Nov2019.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+
+### Turing Way is Expanding!
+
+We are excited to announce that *The Turing Way* is expanding into [a series of books](https://github.com/alan-turing-institute/the-turing-way).
+In addition to its previous focus on reproducibility, this project will aim to capture the vast amounts of knowledge of data science practices in the areas of project design, communication, outreach and ethics.
+Following an open source, community driven approach, we will welcome contributions from anyone from within the Turing community and beyond  who can support researchers and practitioners in learning or improving the "how" of conducting world-leading research in data science.
+The project proposal with further details is available [online](https://github.com/alan-turing-institute/the-turing-way/blob/master/project_management/tps-funding-application-20190429.md).
+
+To plan the next steps, the core team met last month on 11 and 12 October to brainstorm and discuss the next steps in this project.
+The participants of this meeting explored a number of topics in four themed sessions, draft report from which is available on [this HackMD](https://hackmd.io/zVTeKhG2SIiBLam1YtILNg).
```
Comment (KirstieJane) [inline communications/newsletters/newsletter_12_Nov2019.md]: Maybe make the link read only?

File: communications/newsletters/newsletter_12_Nov2019.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+Like our community members, Paola ([GitHub pull request](https://github.com/alan-turing-institute/the-turing-way/issues/648)) and Heidi ([Tweet](https://twitter.com/HeidiBaya/status/1189116056641658880)), have you also wondered how to cite Turing way?
+It is a fair question to ask when referring to an online Open Source project that undergoes several updates in the same week or sometimes on the same day. 
+
+All versions of this book (available under the [cc-by license]((http://creativecommons.org/licenses/by/4.0/legalcode))) can be cited using the DOI: [10.5281/zenodo.3233853](https://doi.org/10.5281/zenodo.3233853).
+This DOI represents all versions, and will always resolve to the latest one.
+Different versions can be found on [Zenodo](https://zenodo.org) and can be individually cited.
+
+(Kirstie's very sorry that she's behind with releasing some of the versions 😩!
+If you attended a book sprint or have written chapters for the book your name will appear in that author list soon.
+Promise!)
```
Comment (KirstieJane) [inline communications/newsletters/newsletter_12_Nov2019.md]: I added this - is this ok?

File: communications/newsletters/newsletter_12_Nov2019.md [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+
+## Acknowledgements and celebrations
+
+Kirstie Whitaker presented a talk at the Open Science Community Nijmegen on 9 October 2019 under the title [*The Turing Way*: A global collaboration for reproducible and collaborative research](https://zenodo.org/record/3478387#.XcmkDZNKjUI), and at NESTA Hack STIR event on 22 October 2019 under the title [*The Turing Way*: A handbook for reproducible research](https://zenodo.org/record/3515937#.XcmkeJNKjUI).
+
+Thanks to several of you for joining Kirstie Whitaker, Yo Yehudi, and Malvika Sharan at the [MozFest House](https://ti.to/Mozilla/mozfesthouse-whyspace/en) to discuss and work on *The Turing Way* specific ideas, and discussing the project with us at the [MozFest 2019](https://www.mozillafestival.org/en/).
+
+![]()
```
Comment (KirstieJane) [inline communications/newsletters/newsletter_12_Nov2019.md]: What were you thinking of adding here? The pic from the steps?

File: communications/newsletters/newsletter_12_Nov2019.md [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+
+The project got shoutouts at [FORCE2019](https://www.force11.org/meetings/force2019) (see this [Tweet by Patricia Hertereich](https://twitter.com/PHerterich/status/1184034991400861696)), at [Goodenough College](https://www.goodenough.ac.uk/) in a [talk by Kirstie](https://zenodo.org/record/3522225#.XcmklpNKjUI), at [SciData19](https://researchdata.springernature.com/users/256983-irene-rodriguez/posts/50480-better-science-through-better-data-2019-scidata19-call-for-lightning-talks-proposal) by Yasemin Turkyilmaz-van der Velden in her talk [Reproducible Research: Why and How?](https://zenodo.org/record/3530485#.XcmopZNKjUI), by Tommy Hall at [Code Mesh London](https://codesync.global/conferences/code-mesh-ldn/) (see this [tweet by Evelina Gabasova](https://twitter.com/evelgab/status/1192804318795182080)) and at [EMBL](http://embl.org) Inspirational Seminar Series in Heidelberg ([talk by Kisrtie](https://zenodo.org/record/3532650#.XcmlAZNKjUI)).
+
+![]()
```
Comment (KirstieJane) [inline communications/newsletters/newsletter_12_Nov2019.md]: What picture here?

File: communications/newsletters/newsletter_12_Nov2019.md (hunk 2 above)
Comment (KirstieJane) [reply communications/newsletters/newsletter_12_Nov2019.md]: And remove link to other hackmd?

File: communications/newsletters/newsletter_12_Nov2019.md (hunk 1 above)
Comment (KirstieJane) [reply communications/newsletters/newsletter_12_Nov2019.md]: Urgh - sorry - I didn't read the PR properly - I can see the images ✨

File: communications/newsletters/newsletter_12_Nov2019.md [hunk 6]
```diff
@@ -56,7 +76,7 @@ Thanks to several of you for joining Kirstie Whitaker, Yo Yehudi, and Malvika Sh
 
 ### Features and Shoutouts
```
Comment (pherterich) [inline communications/newsletters/newsletter_12_Nov2019.md]: Do you want to shout out to Binder for getting CZI money?

File: communications/newsletters/newsletter_12_Nov2019.md (hunk 3 above)
Comment (malvikasharan) [reply communications/newsletters/newsletter_12_Nov2019.md]: Looks good. And Kirstie is doing the best she can so I will remove the part where you don't need to be sorry at all <3

File: communications/newsletters/newsletter_12_Nov2019.md (hunk 4 above)
Comment (malvikasharan) [reply communications/newsletters/newsletter_12_Nov2019.md]: Your tweet with the new contributor who came and chatted with you.

File: communications/newsletters/newsletter_12_Nov2019.md (hunk 6 above)
Comment (malvikasharan) [reply communications/newsletters/newsletter_12_Nov2019.md]: Where does that fit? Maybe where we are mentioning binderhub?

File: communications/newsletters/newsletter_12_Nov2019.md (hunk 5 above)
Comment (malvikasharan) [reply communications/newsletters/newsletter_12_Nov2019.md]: I hope you saw the Tweets (image in the google draft)

File: communications/newsletters/newsletter_12_Nov2019.md (hunk 2 above)
Comment (malvikasharan) [reply communications/newsletters/newsletter_12_Nov2019.md]: Good point. I will do that. I have asked @LouiseABowler to review the report when she has the time.

File: communications/newsletters/newsletter_12_Nov2019.md (hunk 1 above)
Comment (malvikasharan) [reply communications/newsletters/newsletter_12_Nov2019.md]: @KirstieJane - For the book-dash @LouiseABowler and I had a quick chat and we have shortlisted 20-21 February as the most suitable dates (Louise checked your availability and location availability). Can you please approve the date for the newsletter?

Comment (KirstieJane) [reply communications/newsletters/newsletter_12_Nov2019.md]: :+1: Sounds great!

File: communications/newsletters/newsletter_12_Nov2019.md (hunk 5 above)
Comment (KirstieJane) [reply communications/newsletters/newsletter_12_Nov2019.md]: Yep - so sorry about that

File: communications/newsletters/newsletter_12_Nov2019.md (hunk 1 above)
Comment (KirstieJane) [reply communications/newsletters/newsletter_12_Nov2019.md]: Added the dates if you want to review?

File: communications/newsletters/newsletter_12_Nov2019.md (hunk 6 above)
Comment (KirstieJane) [reply communications/newsletters/newsletter_12_Nov2019.md]: Added in this section (acknowledgements and celebrations) - does that make sense? Or are these supposed to be just TW related?

Comment (malvikasharan) [reply communications/newsletters/newsletter_12_Nov2019.md]: I added stuff related to TW, but I suppose it is nice to shoutout to the collaborators and contributors for their successes 😃

File: communications/newsletters/newsletter_12_Nov2019.md (hunk 1 above)
Comment (malvikasharan) [reply communications/newsletters/newsletter_12_Nov2019.md]: Done! looks great!

File: communications/newsletters/newsletter_12_Nov2019.md (hunk 2 above)
Comment (LouiseABowler) [reply communications/newsletters/newsletter_12_Nov2019.md]: The report is looking good @malvikasharan! 🚀 I've left one comment, but that's the only thing I can remember having a discussion about that wasn't already captured  - you've done a great job getting everything into the doc 😄 

I can't edit the file any more so left a couple of typos unfixed, but they're very minor for that amount of content.

Comment (malvikasharan) [reply communications/newsletters/newsletter_12_Nov2019.md]: Thank you @LouiseABowler. My apology for closing the edits without realising that I requested you to edit 🤦‍♂

Comment (KirstieJane) [reply communications/newsletters/newsletter_12_Nov2019.md]: Really awesome work - thank you both of you!!

**Comment by KirstieJane** [comment]
Ok! Added the pictures, and a calendar! https://bit.ly/TuringWayCalendar!! Tell all your friends 😆 

@malvikasharan - click merge if this looks good to you! I'm delighted with it!

**Pull request merged**